TimJohnson/Homework_Week1.js

```diff
@@ -0,0 +1,139 @@
+1a)	"1" == 1; 	true;  using == means equal to and will cast the 1 on the right side of the equation as a string
```

Comment:
Nice job with your explanations. Take a look at the comments below and refer to the homework solutions for more specific answers to these questions.